app/components/viewer/Viewer3D.tsx

```diff
@@ -63,6 +66,7 @@ const Viewer3D = ({ replaysData, timeLineGlobal }: Props): JSX.Element => {
                     lineType={lineType}
                     replayLineOpacity={replayLineOpacity}
                     showGearChanges={showGearChanges}
+                    range={range}
```

Comment:
If this `range` is obtained from a Context anyway, why get the value and then pass it to the next component, if it can just be obtained from the Context in the child component.